src/components/standard/Navigation.js

```diff
@@ -85,7 +78,7 @@ const MainNavbar = () => {
   const Navbar = (props) => {
     return (
       <nav className="navbar">
-        <span className="logo"><img src={MainLogo} height="60px" ></img></span>
+        <span className="logo"><img alt="" src={MainLogo} height="60px" ></img></span>
```

Comment:
Move `height="60px" out to Styles.
Add a valid alt tag like "The Raise Foundation Logo" or something.